### ISSUE TITLE
No module named element tree

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#189]
```
What steps will reproduce the problem?
1. Download XMBC Eden
2. Download Mythbox 1.0.4
3. Launch XBMC
4. Start Mythbox
5 try to select setting

What is the expected output? What do you see instead?

Settings not started

What version of the product are you using? On what operating system?

1.0.4 on mac OS X 10.7

Please provide any additional information below.

Logs are saying the following:
I | bootstrapper.py | MainThread | Line 100 | MythBox 1.0.4 Initialized
D | settings.py | MainThread | Line 152 | Loading settings from 
/Users/bma/Library/Application 
Support/XBMC/userdata/addon_data/script.mythbox/settings.xml
E | util.py | MainThread | Line 307 | (<type 'exceptions.ImportError'>, 
ImportError('No module named elementtree',), <traceback object at 0xe14ba80>)
E | util.py | MainThread | Line 308 | CATCHALL_UI: Caught <type 
'exceptions.ImportError'> exception No module named elementtree on method onInit
Traceback (most recent call last):
  File "/Users/bma/Library/Application Support/XBMC/addons/script.mythbox/resources/src/mythbox/util.py", line 305, in catchall_ui
    return func(*args, **kw)
  File "/Users/bma/Library/Application Support/XBMC/addons/script.mythbox/resources/src/mythbox/ui/home.py", line 88, in onInit
    self.startup()
  File "<string>", line 1, in <lambda>
  File "/Users/bma/Library/Application Support/XBMC/addons/script.mythbox/resources/src/mythbox/ui/toolkit.py", line 196, in window_busy
    result = func(*args, **kwargs)
  File "/Users/bma/Library/Application Support/XBMC/addons/script.mythbox/resources/src/mythbox/ui/home.py", line 168, in startup
    self.goSettings()
  File "/Users/bma/Library/Application Support/XBMC/addons/script.mythbox/resources/src/mythbox/ui/home.py", line 306, in goSettings
    from uisettings import SettingsWindow
  File "/Users/bma/Library/Application Support/XBMC/addons/script.mythbox/resources/src/mythbox/ui/uisettings.py", line 27, in <module>
    from mythbox.advanced import AdvancedSettings
  File "/Users/bma/Library/Application Support/XBMC/addons/script.mythbox/resources/src/mythbox/advanced.py", line 25, in <module>
    from elementtree import ElementTree
ImportError: No module named elementtree
E | util.py | MainThread | Line 307 | (<type 'exceptions.ImportError'>, 
ImportError('No module named elementtree',), <traceback object at 0xe14b620>)
E | util.py | MainThread | Line 308 | CATCHALL_UI: Caught <type 
'exceptions.ImportError'> exception No module named elementtree on method 
onClick
Traceback (most recent call last):
  File "/Users/bma/Library/Application Support/XBMC/addons/script.mythbox/resources/src/mythbox/util.py", line 305, in catchall_ui
    return func(*args, **kw)


```

Original issue reported on code.google.com by `bertrand.marquis` on 4 Dec 2011 at 12:37

**Comment by GoogleCodeExporter** [comment]
```
I solved this by downloading the elementtree plugin for den from:
http://mirror.ufs.ac.za/xbmc/addons/eden-pre/script.module.elementtree/

As this does not seemed to be provided by xbmc by default maybe should this be 
included by mythbox somehow.
```

Original comment by `bertrand.marquis` on 4 Dec 2011 at 12:50

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Eden Beta 2 + MythBox 1.1.0
```

Original comment by `semir.pa...@gmail.com` on 30 Jan 2012 at 12:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Cannot confirm fix. This is reproduceable with xbmx 11.0 on debian squeeze 
(precompiled packages from http://debian.oppserver.net/) and mythbox 1.1.0:

19:00:53 T:140051010803456 WARNING: Create - Unsupported protocol(script) in 
script://
19:00:54 T:140051023390464  NOTICE: -->Python Interpreter Initialized<--
19:00:54 T:140051023390464  NOTICE: MythBox for XBMC
19:00:55 T:140051023390464  NOTICE: MythBox: loggerIniFile = 
/home/mediacenter/.xbmc/addons/script.mythbox/mythbox_log.ini
19:00:55 T:140051023390464  NOTICE: MythBox:Initializing Settings - Error: No 
module named elementtree.ElementTree
19:00:55 T:140051023390464   ERROR: Traceback (most recent call last):
19:00:55 T:140051023390464   ERROR:   File 
"/home/mediacenter/.xbmc/addons/script.mythbox/resources/src/mythbox/bootstrappe
r.py", line 45, in run
19:00:55 T:140051023390464   ERROR:     self.bootstrapSettings()
19:00:55 T:140051023390464   ERROR:   File 
"/home/mediacenter/.xbmc/addons/script.mythbox/resources/src/mythbox/bootstrappe
r.py", line 186, in bootstrapSettings
19:00:55 T:140051023390464   ERROR:     from fanart import FanArt
19:00:55 T:140051023390464   ERROR:   File 
"/home/mediacenter/.xbmc/addons/script.mythbox/resources/src/mythbox/fanart.py",
 line 30, in <module>
19:00:55 T:140051023390464   ERROR:     import tmdb
19:00:55 T:140051023390464   ERROR:   File 
"/home/mediacenter/.xbmc/addons/script.mythbox/resources/lib/themoviedb/tmdb.py"
, line 22, in <module>
19:00:55 T:140051023390464   ERROR:     import elementtree.ElementTree as 
ElementTree
19:00:55 T:140051023390464   ERROR: ImportError: No module named 
elementtree.ElementTree

```

Original comment by `jonas.ha...@gmail.com` on 11 Apr 2012 at 5:02
